docs/src/proposals/page-indexed-transactions.md

```diff
@@ -0,0 +1,235 @@
+# Page-Indexed Transactions
```

Comment:
Why call this "page-indexed"?

Reply:
Because each account can be thought of as a "page" of memory in the page table analogy and those "pages" serve as an account index. It's not the best name, though.. How about "Transactions v2 - Compressed account inputs"